client/src/components/layout/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+export const Navbar = ({ title }) => {
+    return (
+        <div className="navbar bg-primary">
+            <h1>
+                {title}
+            </h1>
+        </div>
+    );
+}

```

Comment:
Are these meant to have `;` after them? Can't remember 🤔 
It works fine as is (because of JS) but I usually like to include semicolons when possible to make it easier to follow.

This is me just being picky. Dealer's choice on what you decide to do re: semicolons (if you do decide to add them, there are a bunch of places in this PR that could have them).